X2WOTCCommunityPromotionScreen/Src/X2WOTCCommunityPromotionScreen/Classes/CPS_UIArmory_PromotionHero.uc

```diff
@@ -1875,3 +1864,153 @@ simulated function AddChildTweenBetween(string ChildPath, String Prop, float Sta
 
 	MC.EndOp();
 }
+
+// Start Issue #28
```

Comment:
I decided to turn around the decision of not fixing log warnings if they are caused by code outside of CPS itself - in this case, parent classes. I copied their functions, sans the parts that caused log warnings, typically by referencing NavHelp or List.